fetools-app/src/components/InputComponents/SliderInput.jsx

```diff
@@ -47,7 +47,7 @@ const SliderInput = forwardRef(function ColorInput(props, ref) {
             defaultValue={defaultValue}
             onChange={onChange}
             type="range"
-            className="block w-full"
+            className="block w-full accent-accent"
```

Comment:
We're going to change the focus/hover color to #7F40BF

```suggestion
            className="block w-full accent-[#7F40BF]"
```